src/OmniSharp/Api/v1/ProjectSystem/OmnisharpController.ProjectSystem.cs

```diff
@@ -24,11 +25,15 @@ public class ProjectSystemController
 
         [HttpPost("/projects")]
         [HttpGet("/projects")]
-        public WorkspaceInformationResponse ProjectInformation()
+        public WorkspaceInformationResponse ProjectInformation(ProjectInformationRequest request)
         {
+            var includeSourceFiles = true;
```

Comment:
I will rename it to `excludeSourceFiles` as well. I wanted to preserve the default behaviour which was to include the source files.